app/controllers/main/routes.py

```diff
@@ -160,7 +160,9 @@ def addInterest(program_id, username):
     username: unique value of a user to correctly identify them
     """
     try:
-        return addUserInterest(program_id, username)
+        success = addUserInterest(program_id, username)
+        if success == True:
```

Comment:
success is already a boolean, so `success == True` is redundant in an `if` condition. Just use your boolean you already have, e.g., `if success:`